clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1103,14 +1109,18 @@ Also, we preserved the following markers, but only for SYS_futex functions:
 - #dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_RETVAL
 - #dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_RETADDR
 
-Finally, every trace has a v2p.textproto file associated to it, which provides a
-plausible virtual to physical mapping of the virtual addresses present in a trace
-for more realistic TLB simulations.  This is a static virtual to physical mapping
-with 2 MB pages. Users can generate different mappings (e.g., smaller page size)
-by modifying this file, or create their own mapping following the same
-v2p.textproto format.
+Every trace has a v2p.textproto and an info.textproto file associated with it.
 
-These traces are supported starting from DynamoRIO 11.3.
+The v2p.textproto file provides a plausible virtual to physical mapping of the virtual
+addresses present in a trace for more realistic TLB simulations.  This is a static
+virtual to physical mapping with 2 MB pages. Users can generate different mappings
+(e.g., smaller page size) by modifying this file, or create their own mapping following
+the same v2p.textproto format.
+
+The info.textproto file provides users with additional, human-readable information about
+a trace.  This file is not intended to be used with any DynamoRIO tool.
```

Comment:
This implies the v2p.textproto can be used with a tool: explicitly mention in the prior paragraph that it can be fed as-is into the `-tool tlb_simulator` of drmemtrace?

Reply:
Done. Added an example.